app/src/androidTest/java/com/sdp/swiftwallet/JavaTest/transactions/FirebaseTransactionHistoryProducerTest.java

```diff
@@ -4,16 +4,13 @@
 //import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
I think I wrote this class and we can delete it now, as we're ignoring the Firebase classes on the CI